src/glass/src/app/core/navigation-bar/navigation-bar.component.scss

```diff
@@ -1,3 +1,13 @@
+@import 'styles.scss';
+
 .glass-navigation-bar {
   min-width: 15rem;
+
+  mat-icon {
+    margin-right: 12px;
+  }
+
+  .mat-list-item:hover {
```

Comment:
Could you please add the following to prevent the jumping of the menu.

![Peek 2021-03-01 15-10](https://user-images.githubusercontent.com/1897962/109508643-5b4c1180-7aa0-11eb-9b8b-507916ba2a08.gif)

![Peek 2021-03-01 15-12](https://user-images.githubusercontent.com/1897962/109508896-9fd7ad00-7aa0-11eb-8a19-e278ff6499ae.gif)

```
diff --git a/src/glass/src/app/core/navigation-bar/navigation-bar.component.scss b/src/glass/src/app/core/navigation-bar/navigation-bar.component.scss
--- a/src/glass/src/app/core/navigation-bar/navigation-bar.component.scss	(revision 021e160305cee9ef2d4872b9c92865b40b687796)
+++ b/src/glass/src/app/core/navigation-bar/navigation-bar.component.scss	(date 1614607704073)
@@ -7,6 +7,9 @@
     margin-right: 12px;
   }
 
+  .mat-list-item {
+    border-left: 5px solid transparent;
+  }
   .mat-list-item:hover {
     border-left: 5px solid $eos-bc-cerulean-900;
   }
```